lib/audit-database/src/migrations/U0003__authentication_method.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE audit_logs
+ADD COLUMN authentication_method JSONB;
+-- Set authentication_method = 'Jwt(Web)' for rows where user_id is not null
+UPDATE audit_logs
+SET authentication_method = '{"method":"Jwt","role":"Web"}'
+WHERE user_id IS NOT NULL;
```

Comment:
Will this always be true for everything prior to this PR? Or will there be example where different methods and roles were used?

Reply:
Everything prior to this PR was either a web token or a system thing, since automation roles are still under feature flag.

(Well, except for a few of our prod workspaces under feature flag, but we're OK with those old audit logs lying to us that it was a webui-initiated change.)